dsp-ci/dsp-gha-runner-instances/values.yaml

```
@@ -30,3 +30,14 @@ runners:
           limits:
             cpu: "4"
             memory: "16Gi"
+  - name: single_cell_portal_core
+    repository: broadinstitute/single_cell_portal_core
+    configOverrides:
```

```suggestion
    repository: broadinstitute/single_cell_portal_core
    minReplicas: 1
    maxReplicas: 1
    configOverrides:
```
https://broadworkbench.atlassian.net/browse/DDO-1797?focusedCommentId=54192

---

Already done, but thanks!